src/features/purchases/screen/orderItem.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import { Image, StyleSheet, Text, View } from 'react-native';
+
+import { LineItem } from 'network/models/product-models';
+
+const OrderItem: React.FunctionComponent<{
+  item: LineItem;
+  index: number;
+  size: number;
+}> = ({ item, index, size }) => {
+  return (
+    <View
+      style={[
+        styles.container,
+        index === 0 && styles.containerFirst,
+        index === size - 1 && styles.containerLast,
+      ]}>
+      <Image
+        source={{ uri: item.product.pictures ? item.product.pictures[0] : '' }}
+        style={styles.image}
+      />
+      <View style={styles.data}>
+        <Text style={styles.name}>{item.product.title}</Text>
+        <Text numberOfLines={1} style={styles.status}>
+          {item.product.state}
+        </Text>
+        <Text style={styles.quantity}>{item.quantity} items</Text>
+        <Text style={styles.price}>{item.totalPrice}</Text>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
```

Comment:
Remember to move these styles to separated file, or moved OrderItem to a components file.